Google.PowerShell/Provider/GoogleCloudStorageProvider.cs

```diff
@@ -1086,7 +1086,8 @@ private IEnumerable<Object> ListChildren(GcsPath gcsPath, bool recurse, bool all
 
         private static IEnumerable<Project> ListAllProjects()
         {
-            ProjectsResource.ListRequest request = ResourceService.Projects.List();
+            Google.Apis.CloudResourceManager.v1.ProjectsResource.ListRequest request =
```

Comment:
Why the full namespace here?

Reply:
Because `Google.Apis.Storage.v1` now also has a `ProjectsResource`.